pallets/living-assets-ownership/src/lib.rs

```diff
@@ -83,16 +83,32 @@ pub mod pallet {
 			OwnerOfCollection::<T>::get(collection_id)
 		}
 
-		fn create_collection(owner: T::AccountId) -> Result<CollectionId, &'static str> {
-			Self::do_create_collection(owner)
+		fn create_collection(
+			owner: T::AccountId,
+		) -> Result<CollectionId, traits::CollectionManagerError> {
+			match Self::do_create_collection(owner) {
+				Ok(collection_id) => Ok(collection_id),
+				Err(err) => match err {
+					Error::CollectionAlreadyExists => {
```

Comment:
As far as I see, there is no possibility to get that error, since collectionId looks for the next available number. Indeed, the code for do_create_collection does never throw a CollectionAlreadyExists error, right?

Reply:
sgtm